### ISSUE TITLE
Directly consume APK push configuration

### ISSUE THREAD

**Issue by mitchhentges** [mozilla-releng/mozapkpublisher#217]
We have a single source of APK push configuration that currently lives in `build-puppet`. This push configuration is read by `pushapkscript` and tells it what channels exist for which tracks, which checks to use for which apps, the package name(s), and more.

This information is also valuable when using `mozapkpublisher` as a CLI, but can't be used at the moment. Instead, we have to pass in a _ton_ of `--skip-check` flags and other information that could be static.

Instead, we should separate APK push configuration from `pushapkscript` configuration (credentials will still sit in `pushapkscript` config and be passed in as CLI args). Then, when doing a `pushapk` task, we should publish the current APK push configuration as an artifact.
Then, if we need to run the `mozapkpublisher` CLI directly, we can download the APK push configuration and point `mozapkpublisher` to it, allowing us to cut down the amount of CLI args that are needed.

TL;DR: cut down on `mozapkpublisher` CLI args by using the APK push config stored in `build-puppet`
